specification/peak-annotation-format/annotation-example-4.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://raw.githubusercontent.com/HUPO-PSI/mzSpecLib/feature/add-annotation-schema/specification/peak-annotation-format/annotation-schema.json",
+    "analyte_reference": 1
```

Comment:
This doesn't seem right. It's an unannotated peak but it has an analyte reference? Wouldn't it be simpler to just not have an object for the unannotated peaks and be done? Otherwise we're back in the same bind with needing a data model for unannotated peaks which overlaps with but is not a subset of the fields needed for annotated peaks. It's a useful idea for spectrum annotation, but it's less useful for the particular problem we're solving here. 

Semantically, this is already the case because we have a peak we can't explain, but we're asserting it is from a specific analyte.

Reply:
Yes, that makes sense. In that case, I'll just drop this example altogether.